packages/protocol/package.json

```diff
@@ -100,6 +100,7 @@
     "nodemailer": "^4.6.8",
     "targz": "^1.0.1",
     "tmp": "^0.1.0",
+    "ts-node": "8.3.0",
```

Comment:
??

Reply:
Scripts in the protocol package require `ts-node`, but it seems like they have been working so far as most of us + CI has it installed globally? I figured I'd add it